### DISCUSSION TITLE
Updating MOOSE

### DISCUSSION THREAD

**Discussion by klaw13** [idaholab/moose#26730] [Q&A Getting Started]
Hi,

I have MOOSE installed on two machines: Mac (M1 chip) and a Linux machine (VM) and am having issues updating both. Before updating they worked fine and I'm unsure how to resolve these issues. Any help is appreciated, thanks!

**Issue with Mac version:** I fail 4 tests when testing the install of MOOSE and am unsure how to resolve these.
<img width="1110" alt="image" src="https://github.com/idaholab/moose/assets/20307070/46454ee9-70c3-4679-9f78-5878a7e31efb">

**Issue with Linux version:** When I try to update conda I get this error. How do I fix this? I tried to revert to an older version, but no matter the version I specify it gives me the same error. I've tried looking in the channels for an active version but could not find anything.

<img width="655" alt="image" src="https://github.com/idaholab/moose/assets/20307070/8f59efad-b9b7-4f63-a9eb-0dbb695044ff">

**Comment by GiudGiud** [comment]
Hello

On the mac:
These are VTK and IGA (which we support only with a newer version of exodus). They are both rather uncommon so I would move on even with these errors. If you run the diagnostics script in moose/scripts I will create an issue and we can address it later.

For linux:
I think this is an old channel.
You need to delete the idaholab channels with `conda config --remove ` + the idaholab channels

then add the new one
```
conda config --add channels https://conda.software.inl.gov/public
```


Guillaume

**Reply by klaw13** [reply]
Thanks! I was able to resolve the Mac version. 

For the Linux version though I'm now getting an error when pulling the most recent version of MOOSE. I have no work done in the heat conduction module. My version of MOOSE hasn't been updated since early-mid 2021 if that is helpful information. 

<img width="527" alt="image" src="https://github.com/idaholab/moose/assets/20307070/4741f898-9390-4f40-91cd-cd17bc557084">

**Reply by GiudGiud** [reply]
The heat conduction module was renamed to heat transfer. 

You can try moving that folder into the heat transfer module, git adding that then continuing the rebase

**Reply by klaw13** [reply]
That works, thank you so much!
